google/cloud/internal/curl_impl.cc

```diff
@@ -43,6 +43,8 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace {
 
 char const* InitialQueryParameterSeparator(std::string const& url) {
+  // Abseil <= 20200923 does not implement StrContains(.., char)
+  // NOLINTNEXTLINE(abseil-string-find-str-contains)
   if (url.find('?') != std::string::npos) return "&";
```

Comment:
My gut was that it would be slower (though probably not so much as to make a difference), I cannot say for sure, but the assembly looks worse:

https://godbolt.org/z/1PrEq4T3T

Reply:
I think you would also say the assembly looks worse for `abs::StrContains(..., string)` compared to `std::string::find(string)`.... but we changed all of those.  🤷‍♂️